.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+---
+version: 2.1
+
+orbs:
+  go: circleci/go@0.2.0
+
+jobs:
+  test:
+    parameters:
+      go_version:
+        type: string
+      run_style:
+        type: boolean
+        default: false
+      use_gomod_cache:
+        type: boolean
+        default: true
+    docker:
+    - image: circleci/golang:<< parameters.go_version >>
+    working_directory: /go/src/github.com/prometheus/exporter-toolkit
+    steps:
+    - checkout
+    - when:
+        condition: << parameters.use_gomod_cache >>
+        steps:
+        - go/load-cache:
+            key: v1-go<< parameters.go_version >>
+    - run: make test
+    - when:
+        condition: << parameters.run_style >>
+        steps:
+        - run: make style
+    - when:
+        condition: << parameters.use_gomod_cache >>
+        steps:
+        - go/save-cache:
+            key: v1-go<< parameters.go_version >>
+    - store_test_results:
+        path: test-results
+
+workflows:
+  version: 2
+  tests:
+    jobs:
+    # Support the last two go relases, as per https://golang.org/dl/
+    - test:
+        name: go-1-14
+        go_version: "1.14"
+    - test:
+        name: go-1-15
+        go_version: "1.15"
+        # Style is only checked against the latest supported Go version.
```

Comment:
I know this is mostly a copy-n-paste from client_golang, but I think we can improve this.

If we split style out as a separate job, we can avoid this. Take a look at [gosnmp CircleCI](https://github.com/gosnmp/gosnmp/blob/c615d988dbf658fcafd2236faa199d624782ce7f/.circleci/config.yml#L53-L67) for an example.